src/runtime/controller.ts

```diff
@@ -17,25 +17,20 @@ export class YfmCutController {
         const cutId = window.location.hash.slice(1);
         const cutNode = document.getElementById(cutId) as HTMLDetailsElement | null;
 
-        if (!(cutNode instanceof HTMLElement)) {
-            return;
-        }
-
-        if (!cutNode.matches(Selector.CUT)) {
+        if (!(cutNode instanceof HTMLElement) || !cutNode.matches(Selector.CUT)) {
             return;
         }
 
         cutNode.classList.toggle(ClassName.OPEN);
         cutNode.setAttribute('open', 'true');
 
-        setTimeout(() => {
-            cutNode.classList.add('cut-highlight');
-            cutNode.scrollIntoView();
-        }, 70);
+        cutNode.classList.add('cut-highlight');
+        cutNode.scrollIntoView();
 
-        setTimeout(() => {
+        cutNode.addEventListener('animationend', function removeHighlight() {
```

Comment:
This style use transition, i guess we need to use `transitionend`

Reply:
@v8tenko @ogonkov 
We also need to check why `70` milliseconds were initially set for adding a class. The DOM may not be immediately available yet. Also, class withdrawal did not occur immediately after the end of the transaction, but after 1000 milliseconds, Why was this done?

Reply:
AbortController seems better solution

Reply:
> @v8tenko @ogonkov 
> We also need to check why `70` milliseconds were initially set for adding a class.

May be we should reflect this in transition description